lib/Minify/ScssCssSource.php

```diff
@@ -0,0 +1,175 @@
+<?php
+use Leafo\ScssPhp\Compiler;
+use Leafo\ScssPhp\Server;
+use Leafo\ScssPhp\Version;
+
+/**
+ * Class for using SCSS files
+ *
+ * @link https://github.com/leafo/scssphp/
+ */
+class Minify_ScssCssSource extends Minify_Source {
+    /**
+     * @var Minify_CacheInterface
+     */
+    private $cache;
+
+    /**
+     * Parsed SCSS cache object
+     *
+     * @var array
+     */
+    private $parsed;
```

Comment:
If the class isn't `final`, Why have private properties in favor of protected ones?